rbs/variables.tf

```diff
@@ -103,3 +103,20 @@ locals {
   stairway_db_name = var.stairway_db_name == "" ? "${local.service}-stairway" : var.stairway_db_name
   stairway_db_user = var.stairway_db_user == "" ? "${local.service}-stairway" : var.stairway_db_user
 }
+
+#
+# Billing account IAM Vars
+#
+# We separate whether the roles/billing.user is set on the RBS SA as its own flag because the
+# Broad SA that runs terraform does not want to have the broad permissions to set modify
+# permissions on the service account. This was done manually instead for the Broad.
+variable "billing_account_id" {
+  type        = string
+  description = "What billing account to assign to the project."
+}
+
+variable "enable_billing_user" {
```

Comment:
can we merge these variables by using a `default = null` on `billing_account_id` and explain that a null billing account id means we won't set the RBS SA as a billing user?
ditto on terra-env

Reply:
Done. Also, I think null doesn't work well, I need to use empty string here. That is my finding in my previous PR to set the optional folder_id